<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserimentoTessera</class>
 <widget class="QWidget" name="InserimentoTessera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>10</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci i dati</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>45</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCognome">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePunti">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineData">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>90</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cognome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Data di nascita</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Punti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inserisci">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
